packages/xstate-react/src/types.ts

```diff
@@ -45,6 +46,12 @@ export interface ActorRefLike<TEvent extends EventObject, TEmitted = any>
   [key: string]: any;
 }
 
+export type ActorRefFrom<
```

Comment:
Wouldnt this also be useful as `xstate`'s export?

Reply:
Not yet; `ActorRef` doesn't exist there yet.

Reply:
Now it does!